### PR TITLE
[DS to LF] Forms : outlined

### PR DESCRIPTION
**Pull request by RipripKipkip**
## Description

Fix outlined textfield/select UI

-----

image

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1900/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1900/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1900/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1900/storybook

File: packages/ng/styles/definitions/select/_select-input.scss [hunk 1]
```diff
@@ -231,16 +231,39 @@
...
+			white-space: normal;
+			margin-bottom: calc(var(--spacings-smallest) * -1);
+		}
+	}
+
+	// Outlined
+	:host-context(.textfield.mod-outlined) {
+		&.textfield-input {
+			&::after {
+				color: var(--palettes-grey-600);
```
Comment (BertrandPodevin) [inline packages/ng/styles/definitions/select/_select-input.scss:250]: You should use ths component specifics vars instead : `--components-field-label-color` ..

File: packages/ng/styles/definitions/select/_select-input.scss [hunk 2]
```diff
@@ -231,16 +231,39 @@
...
-				align-items: center;
+		.lu-select-value {
+			height: auto;
+			align-items: center;
+		}
+
+		.lu-select-display-wrapper {
+			overflow: hidden;
+			white-space: normal;
+			margin-bottom: calc(var(--spacings-smallest) * -1);
```
Comment (BertrandPodevin) [inline packages/ng/styles/definitions/select/_select-input.scss:242]: Why a negative margin here ?

**Review by jeremie-lucca — APPROVED** [review]

**Pull request merged**